example.yml

```
@@ -0,0 +1,230 @@
+        run: |
+          # Set environment (default to production for auto-deploy)
+          if [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
+            ENVIRONMENT="${{ github.event.inputs.environment }}"
+            ACTION="${{ github.event.inputs.action }}"
+            VERSION="${{ github.event.inputs.version }}"
+          else
+            ENVIRONMENT="production"
+            ACTION="deploy"
+            VERSION=""
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)